package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dreamonkey/quasar-app-extension-meta",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
L'aumento di versione lascialo sempre a me, perchè viene fatto in automatico dallo script di deploy da parte di yarn